data/transactions/logic/assembler.go

```diff
@@ -1036,7 +1055,8 @@ var keywords = map[string]OpSpec{
 	"byte": {0, "byte", nil, assembleByte, nil, nil, oneBytes, 1, modeAny, opDetails{1, 2, nil, nil}},
 	// parse basics.Address, actually just another []byte constant
 	"addr": {0, "addr", nil, assembleAddr, nil, nil, oneBytes, 1, modeAny, opDetails{1, 2, nil, nil}},
-}
+	// take a signature, hash it, and take first 4 bytes, actually just another []byte constant
+	"selector": {0, "selector", nil, assembleSelector, nil, nil, oneBytes, 1, modeAny, opDetails{1, 2, nil, nil}}}
```

Comment:
version 1 ? if so why is the code below needed?
```
			if spec.Version > 1 && spec.Version > ops.Version {
				ok = false
			}
```

Reply:
Because originally, I felt pseudo-ops must be version controlled like normal ops.  And, in the general case, they do (if they assemble down to an op that is, itself, versioned).  But this one does not need to be versioned, as it combiles to `byte` which has always existed.

I left the version checking in though, so that the next pseudo-op, which maybe needs the checking, is done right. (For example, we're about to add a pseudo op along with a new `extract` opcode)